packages/website/ts/components/wallet/wallet.tsx

```diff
@@ -128,9 +130,13 @@ const FOOTER_ITEM_KEY = 'FOOTER';
 const DISCONNECTED_ITEM_KEY = 'DISCONNECTED';
 const ETHER_ITEM_KEY = 'ETHER';
 const USD_DECIMAL_PLACES = 2;
+const CRYPTO_COMPARE_MULTI_ENDPOINT = '/pricemulti';
+// Crypto compare recommends requesting no more than once every 10s: https://www.cryptocompare.com/api/?javascript#requests
+const CRYPTO_COMPARE_UPDATE_INTERVAL_MS = 10 * 1000;
```

Comment:
I feel like we should fetch prices once every 10sec on `website-backend` and cache it there. With the current approach, every user will be pinging CryptoCompare directly, which is a lot more load for them. 

We did something similar to this with Coinmarketcap previously, we might be able to revive some of that old code: https://github.com/0xProject/website-backend/commit/f594e086c8c219475b8cb8f304441986898242af
Looks like it's similar to how we cache gas prices from `ethGasStation`.

Reply:
Its a lot more complicated than that because each user has different tokens they are tracking